### PR TITLE
add a count to spop

### PR DESCRIPTION
**Pull request by KostSvitlana**
resolved issue #411

### CODE REVIEW

File: aioredis/commands/set.py [hunk 1]
```diff
@@ -43,9 +43,11 @@ def smove(self, sourcekey, destkey, member):
         """Move a member from one set to another."""
         return self.execute(b'SMOVE', sourcekey, destkey, member)
 
-    def spop(self, key, *, encoding=_NOTSET):
-        """Remove and return a random member from a set."""
-        return self.execute(b'SPOP', key, encoding=encoding)
+    def spop(self, key, count=None, *, encoding=_NOTSET):
+        """Remove and return one or multiple random members from a set."""
+        args = [key]
+        count is not None and args.append(count)
```
Comment (popravich) [inline aioredis/commands/set.py]: This is a nice Python feature but it also a really smelly code style.
What was the reason not to write it in two lines?
Also code coverage for this line is always 100% regardless of the `const` value (branch coverage doesn't work here as well)

**Comment by popravich** [comment]
Thanks for contribution, please fix the above issues and I'll merge it.

File: tests/set_commands_test.py [hunk 2]
```diff
@@ -250,25 +250,39 @@
...
+    for _ in members2:
         test_result = await redis.spop(key, encoding='utf-8')
-        assert test_result in members
+        assert test_result in members2
 
     # make sure set is empty, after all values poped
     test_result = await redis.smembers(key)
     assert test_result == []
 
+    await redis.sadd(key, *members1)
```
[the remark this answers is not in the document]
Comment (KostSvitlana) [reply tests/set_commands_test.py]: I add mark and new tests

**Comment by KostSvitlana** [comment]
Thanks for your help

**Comment by popravich** [comment]
Thanks

**Comment by bukowa** [comment]
Hello! Thanks @KostSvitlana! 

Question: when i install the package via pip this commit is not visible, aioredis version == 1.2.0

**Comment by KostSvitlana** [comment]
> Hello! Thanks @KostSvitlana!
> 
> Question: when i install the package via pip this commit is not visible, aioredis version == 1.2.0

It's because this changes will be in next release

**Comment by Natim** [comment]
Hello, I was going to reopen this issue because I reach this bug, is it possible to release it?

**Pull request merged**